lib/board.rb

```diff
@@ -0,0 +1,133 @@
+# rubocop:disable all
+class TicTacToe
+    attr_accessor :player1,:player2
+        def initialize
+            @board=Array.new(9," ")
+            @player1="X"
+            @player2="O"
+        end
+    
+        WIN_COMBINATIONS=[
+            [0,1,2], 
+            [3,4,5],
+            [6,7,8],
+            [0,4,8],
+            [2,4,6],
+            [0,3,6],
+            [1,4,7],
+            [2,5,8]
+        ]
+    
+        def display_board
+            puts "#{@board[0]} | #{@board[1]} | #{@board[2]}"
+            puts "-----------"
+            puts "#{@board[3]} | #{@board[4]} | #{@board[5]}"
+            puts "-----------"
+            puts "#{@board[6]} | #{@board[7]} | #{@board[8]}"
+        end
+    
+        def input_to_index(string)
+            @choose=string.to_i-1
+        end
+    
+        def move(input_to_index,player)
+            @board[input_to_index]=player
+        end
+    
+        def position_taken?(input_to_index)
+            if @board[input_to_index] == "X" || @board[input_to_index]=="O"
+                true
+            else
+                false
+            end
+        end
```

Comment:
- [ ] This would have the same result as you have
```ruby
def position_taken?(index)
  @board[index] == "X" || @board[index] == "O"
end
```
Another example would be
```ruby
def position_taken?(index)
  ["X", "O"].include?(@board[index])
end
```